src/components/lesson-metadata-modal.js

```diff
@@ -60,12 +78,21 @@ function LessonMetadataModal({ lesson, mode, isVisible, onSave, onClose }) {
     title: lesson.title || t('newLesson'),
     slug: lesson.slug || '',
     language: lesson.language || getDefaultLanguageFromUiLanguage(uiLanguage),
-    startsOn: lesson.schedule?.startsOn ? moment(lesson.schedule?.startsOn) : null
+    startsOn: lesson.schedule?.startsOn ? moment(lesson.schedule?.startsOn) : null,
+    enableSequence: false,
+    sequenceIntervall: LESSON_SEQUENCE_INTERVALL.weekly,
```

Comment:
Some typos here:
- "sequenceIntervall" -> "sequenceInterval"
- "LESSON_SEQUENCE_INTERVALL" -> "LESSON_SEQUENCE_INTERVAL"
- "lessonSequenceIntervallOptions" - "lessonSequenceIntervalOptions"

Reply:
That was denglisch, now it's english!